src/utilities/activity-utils.ts

```diff
@@ -92,3 +92,11 @@ export const enableReportButton = (activity: Activity) => {
   const hasCompletionPage = activity.pages.find((page: any) => page.is_completion);
   return !hasCompletionPage && activity.student_report_enabled;
 };
+
+export const setBackground = (background?:string) => {
+  const gradient = "linear-gradient(to bottom, #fff, rgba(255,255,255,0), rgba(255,255,255,0))";
+  const el = document.querySelector(".app") as HTMLElement;
+  el && ( background? el.style.setProperty("background-image",background) 
+                    : el.style.setProperty("background-image",gradient)
```

Comment:
This function name is a bit vague, but is probably fine (maybe naming it `setAppBackground` tells a little more, but it isn't clear that is used to both set a gradient and a background image - I probably would have made two functions since the functionality here is somewhat different).  Maybe name the argument `backgroundImageURL` to at least help make this clear.

I would clean the spacing up here a tad:
```
  el && ( background
            ? el.style.setProperty("background-image", background) 
            : el.style.setProperty("background-image", gradient)
        );
```

Reply:
I changed it to setAppBackgroundImage. Am wondering though if I need error checking on url that is being passed in?

Reply:
It doesn't seem to break if the URL is bad, we show the color gradient instead.  What happened to the image button in the theme buttons?  We've lost the ability to demo the interactions theme because when we press the Interactions button we now show the star field background.  Seems worth keeping that Interactions button behavior and adding the image button (that was there previously, but now is gone when I pull latest).

Reply:
I would put that image button back in theme-button.tsx:
```
  const handleInteractionsTheme = () => {
    setThemeColors("#414546", "#2f70b0");
    setAppBackgroundImage();
  };
  const handleSetImage = () => {
    setAppBackgroundImage(imageUrl);
  };
```
And
```
<button className="button" onClick={handleSetImage} data-cy="image-button">image</button>
```

Reply:
And I don't think I was clear on the spacing in that function.  I meant something like:
```
export const setAppBackgroundImage = (backgroundImageUrl?: string) => {
  const gradient = "linear-gradient(to bottom, #fff, rgba(255,255,255,0), rgba(255,255,255,0))";
  const el = document.querySelector(".app") as HTMLElement;
  el && ( backgroundImageUrl
          ? el.style.setProperty("background-image", backgroundImageUrl)
          : el.style.setProperty("background-image", gradient)
        );
};
```